app/touch-bar-menu.js

```diff
@@ -3,7 +3,7 @@
 const { TouchBar } = require('electron')
 
 const { TouchBarButton, TouchBarLabel, TouchBarSpacer } = TouchBar
-const MAX_TITLE_LENGTH = 38
+const MAX_TITLE_LENGTH = 39
```

Comment:
With one more button on the TouchBar I would expect the title length to become *shorter* not longer.